library/Vanilla/Formatting/Formats/RichFormat.php

```diff
@@ -61,12 +61,12 @@ public function __construct(Quill\Parser $parser, Quill\Renderer $renderer, Quil
     public function renderHTML(string $content): string {
         try {
             $operations = Quill\Parser::jsonToOperations($content);
-        } catch (FormattingException $e) {
+            $blotGroups = $this->parser->parse($operations);
+            return $this->renderer->render($blotGroups);
+        } catch (\Throwable $e) {
```

Comment:
Is there no way to narrow this down?

Reply:
It is intentional here and it can't be narrowed down because it literally is meant to catch _every single fatal error_ and convert them to a warning. There is __no__ circumstance where we want some bug in post formatting to take down a page.

Reply:
I appreciate the intention of avoiding bad formatting taking down a page, but I think there are better ways to accomplish it than hiding every single error and exception encountered during the formatting step. This is covering up points of failure that might be resolved with defensive coding practices or at least more clearly indicated with specific exceptions.